src/main/java/net/sf/jabref/gui/maintable/MainTableColumn.java

```diff
@@ -63,7 +64,7 @@ public String getDisplayName() {
             return null;
         }
 
-        StringJoiner joiner = new StringJoiner(MainTableFormat.COL_DEFINITION_FIELD_SEPARATOR);
+        StringJoiner joiner = new StringJoiner(JabRefPreferences.COL_DEFINITION_FIELD_SEPARATOR);
```

Comment:
I do not understand this. Why move a constant to JabRefPreferences which is not a preference?